### DISCUSSION TITLE
What do docs mean when referring to file "linking"?

### DISCUSSION THREAD

**Discussion by lonix1** [nuke-build/nuke#995] [Q&A] [answered]
The ["Project Structure"](https://nuke.build/docs/getting-started/setup/#project-structure) docs say this:
 > NUKE organizes these files in different folders as linked files in the build project for you:

and:
> You can deactivate linking of the above files by removing the `NukeRootDirectory` and `NukeScriptDirectory` properties from the build project file  

What is meant here by "linking"? When I read that I automatically thought it was referring to symlinks, but I don't see any in my codebase. And what does deactivation do?

Thanks.

**Comment by matkoch** [comment, marked as answer]
That's about including a couple of files automatically in the project file for easy access

![PDF Expert-nuke-2022-08-16-1QJEBTM0@2x](https://user-images.githubusercontent.com/5005566/184819972-6977442b-b015-4b06-8a34-76cc17802967.png)

**Reply by lonix1** [reply]
I assume that is not for VSCode? I don't have that, and the screenshot is unfamiliar? Rider I assume?

**Reply by matkoch** [reply]
Yes that's Rider. Visual Studio Code only has a folder view.
